source/agora/common/crypto/ECC.d

```diff
@@ -298,9 +298,13 @@ public struct Point
         this.data = data;
     }
 
-    /// Construct a point from its string representation or a `ubyte[]`
-    public this (T) (T param)
-        nothrow @nogc
+    public bool isValid () nothrow @nogc @trusted const
+    {
+        return (crypto_core_ed25519_is_valid_point(this.data[].ptr) == 1);
+    }
+
+    // /// Construct a point from its string representation or a `ubyte[]`
```

Comment:
```suggestion
    /// Construct a point from its string representation or a `ubyte[]`
```

This was removing the symbol from the documentation.

Reply:
Updated